test/cmdlineTests/yul_stack_opt/output

```diff
@@ -39,167 +39,82 @@ object "object" {
 
 
 Binary representation:
-6001808155806002558060035580600455806005558060065580600755806008558060095580600a5580600b5580600c5580600d55808155806002558060035580600455806005558060065580600755806008558060095580600a5580600b5580600c5580600d5580815550
+600180805560028190556003819055600481905560058190556006819055600781905560088190556009819055600a819055600b819055600c819055600d55
 
 Text representation:
     /* "yul_stack_opt/input.yul":98:99   */
   0x01
-  dup1
-  dup2
     /* "yul_stack_opt/input.yul":129:141   */
-  sstore
```

Comment:
Is that the missing redundant store eliminator?

Reply:
Continued in https://github.com/ethereum/solidity/pull/10286#discussion_r621341134.